Covid Lunches/express/express-bootstrap/app.js

```diff
@@ -45,6 +45,10 @@ app.use(express.static(path.join(__dirname, 'public')))
 
 app.get('/', gethomepage)
 
+app.get('/inner', function (req, res) {
```

Comment:
This is good for now. Back end will move this so that it works with the new router though. We may have to deal with that before we merge. Should be an easy 10 minute fix.